tests/test.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
In tests folder, we generally have unit tests in it. So please remove here and place all the code in Readme which is in flow2ml folder(not in global Readme) itself